page/redEvat.jsx

```diff
@@ -0,0 +1,19 @@
+import { 
+    View, 
+    Text, 
+    TextInput,
+    TouchableHighlight,
+    StyleSheet,
+    Alert 
+} from 'react-native'
+import React from 'react'
+
+const redEvat = () => {
+  return (
+    <View>
+        <TextInput />
+    </View>
+  )
+}
+
+export default redEvat
```

Comment:
Following the naming in the other components, the name should be in Pascal case.

```suggestion
export default RedEvat
```

Also the file name.